examples/flows/evaluation/eval-qna-non-rag/gpt_coherence_prompt.jinja2

```diff
@@ -0,0 +1,36 @@
+System:
```

Comment:
Shall we do System -> system, to be consistent with other places?

Reply:
good catch! corrected in the prompt.